osfoffline/views/start_screen.py

```diff
@@ -21,47 +31,68 @@ def __init__(self):
         super().__init__()
         self.start_screen = Ui_startscreen()
 
+    @asyncio.coroutine
     def log_in(self):
-        user_name = self.start_screen.emailEdit.text().strip()
-        password = self.start_screen.passwordEdit.text().strip()
-
-        # assumption: logged in user properly.
-        # assumption: all the fields needed for db are set
-        full_name = user_name
-        osf_id = user_name
-        oauth_token = password
-        # check if user already exists in db
-
+        logging.debug('attempting to log in')
+        personal_access_token = self.start_screen.personalAccessTokenEdit.text().strip()
+        url = settings.API_BASE + '/v2/users/me/'
```

Comment:
We may have to eat it on this pass... but for future methods, we should not put so much request / parsing/ database logic inside a UI callback- this isn't good separation of concerns, and this method is way more complicated than it probably needs to be.